src/vm/tests/test-update.js

```diff
@@ -1480,33 +1480,36 @@ test('add fs /var/tmp/global', function (t) {
             t.ok(false, 'error updating VM: ' + update_err.message);
             t.end();
         } else {
-            VM.load(vm_uuid, function (err, obj) {
-                var field;
+            setTimeout(function () {
```

Comment:
Can we get a comment somewhere saying why we're doing this? In particular it's a workaround?

Reply:
Will do, probably later tonight